Lib/test/test_ctypes/test_dlerror.py

```diff
@@ -58,11 +58,14 @@ def test_null_dlsym(self):
         import subprocess
         import tempfile
 
-        retcode = subprocess.call(["gcc", "--version"],
-                                  stdout=subprocess.DEVNULL,
-                                  stderr=subprocess.DEVNULL)
-        if retcode != 0:
+        try:
+            retcode = subprocess.call(["gcc", "--version"],
+                                      stdout=subprocess.DEVNULL,
+                                      stderr=subprocess.DEVNULL)
+        except FileNotFoundError:
```

Comment:
Let's also catch permission errors and such, as we do in `test_find`:
```suggestion
        except OSError:
```